src/Archive.php

```diff
@@ -340,7 +340,12 @@ protected function send($data)
 
 		$this->need_headers = false;
 
-		fwrite($this->output_stream, $data);
+		do
+		{
+			$result = fwrite($this->output_stream, $data);
+			$data = substr($data, $result);
+			fflush($this->output_stream);
```

Comment:
Based on the PHP documentation [fflush](https://secure.php.net/manual/en/function.fflush.php) is designed for file pointers. Is this going to be a problem with regular streams?

Reply:
Let me eat my own words, looks like streams accept it. http://php.net/manual/en/streamwrapper.stream-flush.php